.github/workflows/integration-tests.yml

```diff
@@ -55,3 +55,47 @@ jobs:
             -e "operator_pipeline_image_tag=${{ github.sha }}"
             -e "suffix=${{ steps.prepare.outputs.suffix }}"
             -v
+
+  integration-tests-community-operators:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Prepare
+        id: prepare
+        run: |
+          echo "suffix=${GITHUB_SHA::7}" >> $GITHUB_OUTPUT
+
+      - name: Install dependencies
+        run: |
+          # Install python dependencies
+          pip install --user openshift pygithub
+
+          echo "${{ secrets.VAULT_PASSWORD }}" > "$HOME"/.vault-password
+
+          mkdir -p "$HOME"/.kube
+          ansible-vault decrypt \
+            --vault-password-file "$HOME"/.vault-password \
+            --output "$HOME"/.kube/config \
+            ansible/vaults/integration-tests/ci-pipeline-kubeconfig
+
+          mkdir -p "$HOME"/.ssh
+          ansible-vault decrypt \
```

Comment:
Same here.

Reply:
Yep, you are right, good catch, removing.

Reply:
Returned back, due to findings summarized in comment.